docs/review/api/alfa-aria.api.md

```diff
@@ -103,8 +103,9 @@ export namespace DOM {
     hasNonEmptyAccessibleName: typeof dom.hasNonEmptyAccessibleName, // (undocumented)
     hasRole: typeof dom.hasRole, // (undocumented)
     isIgnored: typeof dom.isIgnored, // (undocumented)
```

Comment:
Is `isIgnored` still needed now that `isIncludedInTheAccessibilityTree` exists?

Reply:
More or less. There are still a few cases where we use it. We could of course use `not(isIncludedInTheAccessibilityTree)` instead, but I'm not sure this is clearer… So leaving it for now 😃